### PR TITLE
Accordion - icon is black by default, with hover effect to core-purple

### PR DESCRIPTION
**Pull request by oyvindbergbouvet**

### CODE REVIEW

File: src/molecules/AccordionList/AccordionList.pcss [hunk 1]
```diff
@@ -88,9 +88,6 @@ p + .accordion-list {
       background-color: hsl(0, 0%, 97%);
       outline: none;
     }
-    &:focus + .accordion__header-icon *,
-    &:active + .accordion__header-icon * {
-    }
```
Comment (oyvindbergbouvet) [inline src/molecules/AccordionList/AccordionList.pcss:93]: Removed - its empty

File: src/molecules/AccordionList/AccordionList.pcss [hunk 2]
```diff
@@ -144,10 +141,15 @@ p + .accordion-list {
         padding-top: 0.2rem;
       }
     }
+
+    &:hover > .accordion__icon *,
+    &:hover + .accordion__header-icon * {
+      fill: var(--core-purple);
```
Comment (oyvindbergbouvet) [inline src/molecules/AccordionList/AccordionList.pcss:147]: hover-effect on the "whole accordion button" (the whole heading of the accordion)
- both icons within it, gets same color

File: src/molecules/AccordionList/AccordionList.pcss [hunk 3]
```diff
@@ -144,10 +141,15 @@ p + .accordion-list {
...
+    }
   }
 
   &__header-icon {
-    fill: var(--core-purple);
+    fill: var(--black);
```
Comment (oyvindbergbouvet) [inline src/molecules/AccordionList/AccordionList.pcss:152]: default icon color for the 'arrow' in the accordion

File: src/molecules/AccordionList/AccordionList.pcss [hunk 4]
```diff
@@ -160,7 +162,7 @@ p + .accordion-list {
   }
 
   &__icon {
-    fill: var(--core-purple);
+    fill: var(--black);
```
Comment (oyvindbergbouvet) [inline src/molecules/AccordionList/AccordionList.pcss:165]: default icon color, for the 'left icon' which is not the arrow icon

File: src/molecules/AccordionList/AccordionList.pcss [hunk 5]
```diff
@@ -209,10 +211,18 @@ p + .accordion-list {
         background-color: var(--white);
         color: var(--core-purple);
       }
+      
+      &:focus > .accordion__icon *,
+      &:active > .accordion__icon *,
       &:focus + .accordion__header-icon *,
```
Comment (oyvindbergbouvet) [inline src/molecules/AccordionList/AccordionList.pcss:217]: Adding same effect for 'icon' as the 'arrow icon' (which is header-icon)

File: src/molecules/AccordionList/AccordionList.pcss [hunk 6]
```diff
@@ -209,10 +211,18 @@ p + .accordion-list {
...
       &:active + .accordion__header-icon * {
         fill: var(--core-purple);
       }
+
+      &:hover > .accordion__icon *,
+      &:hover + .accordion__header-icon * {
+        fill: var(--core-purple);
+      }
```
Comment (oyvindbergbouvet) [inline src/molecules/AccordionList/AccordionList.pcss:225]: Same color on hovering an open accordion

**Review by imerbengt — APPROVED** [review]

**Pull request merged**